examples/k8s-bookinfo.yaml

```diff
@@ -56,8 +56,10 @@ spec:
           value: http://$(CONTROLLER_SERVICE_HOST):$(CONTROLLER_SERVICE_PORT)
         - name: A8_DISCOVERY_BACKEND
           value: kubernetes
-        - name: A8_SERVICE
-          value: gateway:none
+        - name: A8_KUBERNETES_POD_NAME
```

Comment:
Are you saying that nginx has been launched with `A8_SERVICE=:`? Can you provide some more debug info so I can recreate?

For the gateway, nginx should be launched with `A8_SERVICE=gateway:service=gateway`, which is the k8s service implemented by this pod, along with its pod labels. If there's any dependency in our nginx code on the presence of a `none` tag, then that should be fixed.